### PR TITLE
Ansible Full Setup

### PR DESCRIPTION
**Pull request by jamiecook**

### CODE REVIEW

File: ops/ansible/roles/common/tasks/bootstrap.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  failed_when: false
+  register: check_ubuntu_user
+  tags: bootstrap
+
+- block:
+  - name: Add the user 'ubuntu' with a specific groups
+    ansible.builtin.user:
+      name: ubuntu
+      groups: sudo
+      password: "$1$IveBeenA$UlFfzeZc0GDrihfkKvKdY."
```
Comment (Sleepingwell) [inline ops/ansible/roles/common/tasks/bootstrap.yaml:14]: Not knowing much about ansible... I presume the $ signs are some sort of interpolation?

Comment (jamiecook) [reply ops/ansible/roles/common/tasks/bootstrap.yaml:20]: as discussed - they are encoding of the salt used to encrypt this

**Comment by jamiecook** [comment]
@Sleepingwell  - this is ready to merge. I'd use a squash merge as i'm pretty loose with my commits

**Review by Sleepingwell — APPROVED** [review]

**Comment by Sleepingwell** [comment]
Whoops... only just saw you advice on the squash merge.

**Pull request merged**